packages/zql/src/mutate/mutator-registry.ts

```diff
@@ -148,22 +141,28 @@ export function defineMutators<S extends Schema, C>(
  * Gets a Mutator by its dot-separated name from a MutatorRegistry.
  * Returns undefined if not found.
  */
-export function getMutator(
-  registry: AnyMutatorRegistry,
+export function getMutator<
+  MD extends AnyMutatorDefinitions,
+  TSchema extends Schema = DefaultSchema,
+>(
+  registry: MutatorRegistry<MD, TSchema>,
   name: string,
-): AnyMutator | undefined {
+): FromMutatorTree<MD, TSchema> | undefined {
```

Comment:
I do not understand what's going on here. It looks like this returns the entire mutator tree, but `getMutator` just returns a single mutator.